valence-rs/valence
# WIP: building example

This adds a new example to demonstrate how to implement block breaking/placing.

In order for this to work, I also had to:
- update definition for UpdateCreativeModeSlot

I'm putting this up so people know it exists, I'll finish it up later.

# Test plan

1. run `cargo r -r --example building`
2. start minecraft and join localhost
3. See that breaking and placing blocks works

NOTE: We don't keep track of inventories yet, so no matter what block you place it's going to be dirt instead.
